src/components/AnonymousChat/child/ChatReplyPreview.tsx

```diff
@@ -105,6 +126,25 @@ const styles = StyleSheet.create({
   containerDismiss: {
     marginLeft: 5,
     padding: 5
+  },
+  mediaContainer: {
+    width: 44,
+    height: 44,
+    position: 'absolute',
+    right: 0,
+    top: 0
+  },
+  mediaImage: {
+    width: '100%',
+    height: '100%'
+  },
+  mediaOverlay: {
+    position: 'absolute',
+    backgroundColor: 'rgba(0,0,0,0.5)',
+    width: '100%',
+    height: '100%',
+    justifyContent: 'center',
+    alignItems: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles defined here are static and do not depend on any state or props. It's better to move these styles to a StyleSheet to improve performance.
```diff
-  mediaContainer: {
-    width: 44,
-    height: 44,
-    position: 'absolute',
-    right: 0,
-    top: 0
-  },
-  mediaImage: {
-    width: '100%',
-    height: '100%'
-  },
-  mediaOverlay: {
-    position: 'absolute',
-    backgroundColor: 'rgba(0,0,0,0.5)',
-    width: '100%',
-    height: '100%',
-    justifyContent: 'center',
-    alignItems: 'center'
-  }
+  mediaContainer: styles.mediaContainer,
+  mediaImage: styles.mediaImage,
+  mediaOverlay: styles.mediaOverlay
```
In your StyleSheet:
```javascript
StyleSheet.create({
  // ...
  mediaContainer: {
    width: 44,
    height: 44,
    position: 'absolute',
    right: 0,
    top: 0
  },
  mediaImage: {
    width: '100%',
    height: '100%'
  },
  mediaOverlay: {
    position: 'absolute',
    backgroundColor: 'rgba(0,0,0,0.5)',
    width: '100%',
    height: '100%',
    justifyContent: 'center',
    alignItems: 'center'
  }
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->